server/preview/index.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const ctrl = require('./preview.controller');
+const router = require('express').Router();
+
+router
+  .post('/', ctrl.getPreviewUrl)
+  .get('/:id', ctrl.getActivity);
```

Comment:
Why did you remove cors control for `POST` and `OPTIONS` here?

Reply:
By mistake.